### PR TITLE
COMMERCE-10008 Remove search and pagination from the data-sets in the Account's Channel Defaults

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
https://issues.liferay.com/browse/COMMERCE-10008

Resent from #3573 

@CrescenzoRegaLF

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro [hunk 1]
```diff
@@ -1,5 +1,386 @@
...
+				-H 'Content-Type: application/json' \
+				-d '{
+					"channelId": ${commerceChannelId},
+					"classPK": ${billingAddressId},
+					"overrideEligibility": "${overrideEligibility}",
+					"priority": ${priority}
+				}'
+		''';
+
+		com.liferay.poshi.runner.util.JSONCurlUtil.post(${curl});
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro:39]:
```suggestion
		JSONCurlUtil.post(${curl});
```

Please apply it to all occurrences

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro [hunk 2]
```diff
@@ -1,5 +1,386 @@
...
+
+		var baseURL = ${baseURL};
+
+		if (!(isSet(baseURL))) {
+			var baseURL = PropsUtil.get("portal.url");
+		}
+
+		var accountEntryId = JSONAccountEntryAPI._getAccountEntryId(accountEntryName = ${accountEntryName});
+		var commerceChannelId = CommerceJSONChannelsAPI._getCommerceChannelIdByName(channelName = ${channelName});
+		var commerceCurrencyId = CommerceJSONCurrenciesAPI._getCommerceCurrencyIdByCode(code = ${code});
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro:90]: Can you change the variable `code` in this block to `currencyCode`?

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro [hunk 3]
```diff
@@ -1,5 +1,386 @@
...
+
+		var baseURL = ${baseURL};
+
+		if (!(isSet(baseURL))) {
+			var baseURL = PropsUtil.get("portal.url");
+		}
+
+		var accountEntryId = JSONAccountEntryAPI._getAccountEntryId(accountEntryName = ${accountEntryName});
+		var commerceChannelId = CommerceJSONChannelsAPI._getCommerceChannelIdByName(channelName = ${channelName});
+		var termsAndConditionsId = CommerceJSONTermsAndConditionsAPI._getCommerceTermsAndConditionsIdByName(termsAndConditionsName = ${termsAndConditionsName});
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONAccountsAPI.macro:127]:
```suggestion
		var termId = CommerceJSONTermsAndConditionsAPI._getCommerceTermsAndConditionsIdByName(termsAndConditionsName = ${termsAndConditionsName});
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase [hunk 4]
```diff
@@ -657,4 +677,1210 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-11227 | Can see pagination working properly for billing addresses table in the account channel defaults"
+	@priority = 3
+	test CanSeePaginationWorkingProperlyForBillingAddressesTable {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase:682]:
```suggestion
	test CanAssertChannelDefaultsBillingAddressesTablePagination {
```
Please apply this change in all test names

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase [hunk 5]
```diff
@@ -657,4 +677,1210 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-11227 | Can see pagination working properly for billing addresses table in the account channel defaults"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase:680]:
```suggestion
	@description = "COMMERCE-11227 | Can use pagination for Billing Address table in the account channel defaults"
```
Please apply this change to all similar occurrences

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase [hunk 6]
```diff
@@ -657,4 +677,1210 @@ definition {
...
+				AssertElementNotPresent(
+					key_firstColumnValue = "Test Channel ${num}",
+					key_secondColumnValue = "",
+					key_table = "ShippingOption",
+					locator1 = "CommerceEntry#TABLE_ASSERT_TWO_COLUMNS_EDIT_BUTTON");
+			}
+		}
+
+		task ("When the admin goes to the next page") {
+			Click(locator1 = "ACUtils#NEXT_PAGES");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannelsDefaultPaymentMethods.testcase:1862]: Same as above

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@22LucaGiannotta22 I've left a couple of comments for you. Please have a look at them.
Also, as a general thing, the tests are more or less covering only the click of the next page, and the check for the previous page or an exact page are not present.

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**